sandbox/namespace_linux.go

```diff
@@ -44,11 +44,16 @@ func GenerateKey(containerID string) string {
 
 // NewSandbox provides a new sandbox instance created in an os specific way
 // provided a key which uniquely identifies the sandbox
-func NewSandbox(key string) (Sandbox, error) {
-	return createNetworkNamespace(key)
+func NewSandbox(key string, create bool) (Sandbox, error) {
```

Comment:
Same here. `create` is misleading. can you please name it more appropriately such as `createNS` ?